doc/source/myrocks/zenfs.rst

```diff
@@ -13,99 +13,191 @@ a Linux kernel implementation which supports NVMe Zoned Namespaces. The kernel
 must be configured with `zone block device support
 enabled <https://zonedstorage.io/linux/config/#kernel-configuration>`__.
```

Comment:
Note that the URL has changed to https://zonedstorage.io/docs/linux/config

Reply:
Fixed.